tests/yaml/install/circular.yml

```diff
@@ -0,0 +1,45 @@
+base:
+  available:
+    - A 1.0.0; depends B == 1.0.0
+    - B 1.0.0; depends C == 1.0.0
+    - C 1.0.0; depends D == 1.0.0
+    - D 1.0.0; depends A == 1.0.0
+
+cases:
+# NOTE: Do we want to check the order?
```

Comment:
This PR does not do so right now - mainly due to the fact that the system does not actually parse the output of pip and instead looks in `result.files_created` to determine what got installed.